apps/dashboard/src/lib/utils/functions/IsSubmissionEarly.spec.js

```diff
@@ -0,0 +1,30 @@
+import isSubmissionEarly from './isSubmissionEarly';
+
+describe('isSubmissionEarly', () => {
+  test('should return true when createdAt is earlier than dueDate', () => {
+    const due_date = Date.now();
```

Comment:
Please use camel case not snake case. Thanks.

We are following the Airbnb js convention: https://github.com/airbnb/javascript?tab=readme-ov-file#variables